03/app.js

```diff
@@ -1,3 +1,13 @@
-export default function randomNumber(min, max) {
+export default function randomNumber(min=0,max=0) {
+    if(typeof min!=='number' || typeof max!=='number') {
+        throw new Error('Property have to be a number');
+    }
```

Comment:
Warto sprawdzić każdą z osobna - w ten sposób możemy określić dokładny komunikat (która jest nie poprawna) i dokładnie to też przetestować.